src/ui/Views/Shared/_Alerts.cshtml

```diff
@@ -0,0 +1,19 @@
+@{
+  var type = (string)TempData.Get("MessageType");
+  var title = (string)TempData.Get("MessageTitle");
+  var body = (string)TempData.Get("MessageBody");
+}
+
+@if (type != null)
+{
+  <div class="govuk-@type-summary" role="alert">
+    <h3 class="govuk-heading-m govuk-@type-summary__title">@title</h3>
```

Comment:
Is h3 the right heading level?

Reply:
Given that its below the <h1> I think i'll make it a <h2> 👍